tools/update_vendor.sh

```diff
@@ -9,6 +9,9 @@ if [ ! -d "vendor" ]; then
     exit 1
 fi
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+TMP_DIR="${TMP_DIR:-$(mktemp -d ${KUBE_ROOT}/tools/update-vendor.XXXX)}"
```

Comment:
Where is this variable used?

Reply:
Nice Catch! Previously I added this var for more checkings in update_vendor.sh. Later I decided to add more checkings on a separate pr. Updated.